request-postgres/entrypoint.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+complete() {
+    echo "done"
+    [[ $KEEP_ALIVE -eq 1 ]] || exit 0
+    echo "Keep alive active!"
+    while :; do
+        sleep 1
+    done
```

Comment:
Under what circumstances would we enable `KEEP_ALIVE`?

Reply:
This option provides the means to keep a job / container alive so you can execute a shell (if need be) do debug anything. Without it, the container simply exits